### PR TITLE
Gauss plot2

### PR DESCRIPTION
**Pull request by karm-patel**
## Checklist:

- [ ] Performed a self-review of the code
- [ ] Tested on Google Colab.

### CODE REVIEW

File: notebooks/gauss_plot.ipynb [hunk 1]
```diff
@@ -0,0 +1,150 @@
+{
```
Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <p>Remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # import superimport # Deprecated
</pre><p>Remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    except ModuleNotFoundError:
</pre><p>Let us not use  <code>ModuleNotFoundError</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Set this to False manually if running this notebook locally in stand-alone mode (without cloning the repo).
</pre><p>No need for these lines now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        from latexify import latexify, format_axes
</pre><p>I have removed format_axes function from latexify. We use sns.despline everywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.        latexify(n_figures=2, fig_height=1.5)
</pre><p>Syntax is changed now. Check docstring to know more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Plot pdf and cdf of standard normal
</pre><p>Reuse some code by creating a function. There is a repeatation of code here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    X = jnp.linspace(-3, 3, 500)
</pre><p>X should be small.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (patel-zeel) [inline notebooks/gauss_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    plt.show()
</pre><p>Check if this line helps in our workflow (better output to jupyter-book?) else we should simply remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: notebooks/quantile_plot.ipynb [hunk 2]
```diff
@@ -0,0 +1,198 @@
+{
```
Comment (patel-zeel) [inline notebooks/quantile_plot.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #50.    sns.despine()
</pre><p>This should be called before pml.savefig else it wont work during saving.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: notebooks/gauss_plot.ipynb (hunk 1 above)
Comment (karm-patel) [reply notebooks/gauss_plot.ipynb:1]: <p>Done</p>

**Pull request closed without merging**